README.md

```diff
@@ -17,10 +17,15 @@ For example:
 Before running the tests, first install the dependencies:
 
 ```bash
+bundle config set --local with 'test'
```

Comment:
What does this do?

Reply:
`bundle install --with=test` produces that advice, so I thought it would be useful to add it to the instructions directly, because testing will require the installation of [that gem group](https://github.com/exercism/ruby-test-runner/blob/9345867127a29b3aebcdb4b9d077a4b5f01bb9a1/Gemfile#L16-L23).